src/main/java/com/witherview/selfPractice/QuestionList/SelfQuestionListApiController.java

```diff
@@ -0,0 +1,62 @@
+package com.witherview.selfPractice.QuestionList;
+
+import com.witherview.database.entity.QuestionList;
+import com.witherview.exception.ErrorCode;
+import com.witherview.exception.ErrorResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpSession;
+import javax.validation.Valid;
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+public class SelfQuestionListApiController {
+    private final SelfQuestionListService selfPracticeService;
+    private final long userId = 1; // 임시
+
+    // 질문 리스트 등록
+    @PostMapping("/self/questionList")
```

Comment:
요청시의 Content-Type이 Json일 경우에, Accept 헤더 값이 Json일 경우에만 컨트롤러를 매핑시켜주는 것은 어떨까요?

```suggestion
    @PostMapping(path = "/self/questionList", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
```